### ISSUE TITLE
support byte-range request to object

### ISSUE THREAD

**Issue by ywang19** [intel-cloud/cosbench#269]
some scenarios need access byte range of objects, hope to see this kind of features.

**Comment by tsg-** [comment]
Is the first target openstack swift?

**Comment by ywang19** [comment]
yes, will use swift as the starting point to probe the implications.

**Comment by gonewest818** [comment]
+1     we also use byte ranges a fair amount, would be helpful to benchmark the performance.

**Comment by hrast01** [comment]
This has come up as a concern for us as well.  The rest of the tool fits our use cases perfectly, this is the only test case we're missing.
